charts/dsa-device-plugin/values.yaml

```diff
@@ -0,0 +1,9 @@
+dsa:
+  metadata:
+    name: dsadeviceplugin-sample
+  spec:
+    image: intel/intel-dsa-plugin:0.24.0
```

Comment:
looks like the samples do not cover the `initImage` but that's desired/required for both DSA/IAA.

Also, please also follow what we added in #9. https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/deployments/nfd/overlays/node-feature-rules/node-feature-rules.yaml

Reply:
Do you mean to add node feature rules?

Reply:
yes